# Bioinformatic_tools
## LncRNA
CPC2: https://github.com/gao-lab/CPC2_standalone/releases/tag/v1.0.1<br>
PLEK: https://sourceforge.net/projects/plek<br>
CNCI: https://github.com/www-bioinfo-org/CNCI<br>
CPAT: https://github.com/liguowang/cpat   训练模型https://sourceforge.net/projects/rna-cpat/files/v1.2.2/prebuilt_model/<br>
LGC:  https://ngdc.cncb.ac.cn/biocode/tools/BT000004<br>

## Fastq Quatity control
fastp: https://github.com/OpenGene/fastp<br>

## Download software
sratoolkit: https://github.com/ncbi/sra-tools/wiki/01.-Downloading-SRA-Toolkit<br>

## Database
SPEED2: https://speed2.sys-bio.net  (SPEED2 is a signaling pathway enrichment analysis tool)<br>
Interpro: https://www.ebi.ac.uk/interpro/<br>

## Function annotation
Interproscan: https://github.com/ebi-pf-team/interproscan<br>
eggNOG： https://github.com/eggnogdb/eggnog-mapper<br>

## gene prediction
https://github.com/harvardinformatics/GenomeAnnotation<br>
Augustus: https://github.com/Gaius-Augustus/Augustus<br>

## ChIP-seq
WiggleTools：https://github.com/Ensembl/WiggleTools<br>
ChIP-seq-analysis：https://github.com/crazyhottommy/ChIP-seq-analysis<br>
Intro-to-ChIPseq：https://hbctraining.github.io/Intro-to-ChIPseq  https://github.com/hbctraining/Intro-to-ChIPseq-flipped<br>

## Sequence align
hmmer: http://hmmer.org/<br>

## multiple sequence align
muscle： https://github.com/rcedgar/muscle<br>
MAFFT: https://mafft.cbrc.jp/alignment/software<br>
Kalign: https://github.com/timolassmann/kalign<br>
Clustal Omega: http://www.clustal.org/omega<br>
colections: https://www.ebi.ac.uk/Tools/msa/<br>

## TCGA数据库相关
UCSC Xena：http://xena.ucsc.edu/ TCGA数据下载<br>
GEPIA: http://gepia2.cancer-pku.cn/#index TCGA数据在线挖掘，差异基因分析、生存分析等<br>
### c-Bioportal: www.cbioportal.org<br>
整合和简化了包括TCGA，ICGC以及GEO等多个癌症基因组数据库的内容，提供友好可视化的界面，可供下载
主要展示基因的somatic 突变谱，拷贝数变化，mRNA&miRNA表达量变化，DNA甲基化以及蛋白质表达的情况，并结合患者的临床资料，展示了KM生存曲线<br>
### OncoLnc: www.oncolnc.org<br>
这是一个整合了TCGA的各种RNA数据和患者临床数据，提供生存分析的网站，灰常简单好用<br>
### MEXPRESS:mexpress.be/about<br>
整合了TCGA中的DNA甲基化，表达量及临床数据，主要用来探索甲基化，基因表达和临床表型之间的关联<br>
